modules/core/src/lib/deck.ts

```diff
@@ -613,6 +614,20 @@ export default class Deck {
     return this._pick('pickObjects', 'pickObjects Time', opts);
   }
 
+  addWidget(
+    widget: Widget,
+    opts?: {
```

Comment:
This `opts` abbreviation has entrenched itself hard... Not sure pushing back on it is worthwhile:
```suggestion
    options?: {
```